jobs/influxdb/spec

```diff
@@ -11,6 +11,9 @@ packages:
 properties:
   influxdb.database:
     description: "database name in influxdb to create"
+    default: "/var/vcap/store/influxdb"
+  influxdb.datadir:
+    description: "directory where influxdb will keep its data"
```

Comment:
This should probably have a default, since the templates require it.

Actually looks like the default value above should have been for this property.